src/validations/html/htmlInspector.js

```diff
@@ -38,8 +39,37 @@ const errorMap = {
       },
     };
   },
+  'invalid-list-children': (error) => {
+    const tag = error.context.tagName.toLowerCase();
+
+    return {
+      reason: 'invalid-list-children',
+      payload: {
+        tag,
+        children: 'li',
+      },
+    };
+  },
 };
 
+function invalidListChildrenValidator(listener, reporter) {
```

Comment:
I would name this something a little more specific, e.g. `noListsWithTextChildrenValidator`

Reply:
Done